src/middlewared/middlewared/plugins/system.py

```diff
@@ -240,9 +240,17 @@ async def do_update(self, data):
         hardware.
 
         When `syslogserver` is defined, logs of `sysloglevel` or above are sent.
+
+        `consolemsg` is a deprecated attribute and will be removed in further releases. Please, use `consolemsg`
+        attribute in the `system.general` plugin.
         """
+        consolemsg = None
+        if 'consolemsg' in data:
```

Comment:
Wonder if it makes sense to generate a DeprecateWarning

Reply:
To the middleware log? What for, if it is a client issue?

Reply:
We do that elsewhere. Developers may look at our logs. It sounds better than nothing.